### DISCUSSION TITLE
blog/2024/04/last-post-changing-blog-website-link/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#11] [Giscus]
# blog/2024/04/last-post-changing-blog-website-link/

This will be my last post on here, just letting all of you know that I will be moving the website to my new domain that I just purchased: https://blog.liluzibird.dev.
If you’re reading this from the new site, yay, thank you. If there are any kinks with the new site, please let me know.
I hope to see all of you over here!

https://blog.liluzibird.dev/blog/2024/04/last-post-changing-blog-website-link/

**Comment by liluzibird** [comment]
Website migration comments test
